resources/account.rb

```diff
@@ -35,6 +35,7 @@
 attribute :ssh_keys,      :kind_of => [Array,String], :default => []
 attribute :groups,        :kind_of => [Array,String], :default => []
 attribute :ssh_keygen,    :default => nil
+attribute :data_bag,      :kind_of => String, :default => 'ssh_public_keys'
```

Comment:
It seems like this name is a bit ambiguous regarding its functionality. How do you feel about changing to something like `ssh_pubkey_data_bag` for clarity?